spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    name "Eliot Sykes"
```

Comment:
Spelling mistake if this is a reference to the Elliott Sykes in Knight Rider's [Knight & Knerd episode](http://www.imdb.com/title/tt0620831/)